template/canvas.html

```diff
@@ -222,23 +222,35 @@ <h1 class="color_change main-heading">PixelVibe</h1>
 
 colorInput.addEventListener("change", () => {
   brushColor = colorInput.value
+  eraser.classList.remove("selected");
+  brush.classList.add("selected");
+  canvas.classList.remove("erasing");
 })
 
 //Color palette
 palette.addEventListener('click', (e) => {
   const boxShadow = 'inset 0 0 6px #616161';
   brushColor = e.target.style.backgroundColor;
-  prevBrushColor=brushColor;
 
```

Comment:
@N-Shar-ma please don't remove this line . i Think due to this , fill feature is not working properly

Reply:
Done